packages/web-app-files/src/components/Collaborators/CollaboratorsEditOptions.vue

```diff
@@ -1,17 +1,28 @@
 <template>
   <oc-grid gutter="small" child-width="1-1">
-    <roles-select
-      mode="collaborators"
-      :roles="roles"
-      :selected-role="role"
-      @roleSelected="selectRole"
-    />
+    <label class="oc-label">
+      {{ $gettext('Role') }}
+      <oc-select
```

Comment:
Can you move the `oc-select` out of the label and use `for="files-collaborators-role-button"` on the label instead? Don't know if this is strictly necessary for a11y, but we have it like that now everywhere else, that label and input are in parallel in the DOM, not wrapped anymore.

Reply:
> Don't know if this is strictly necessary for a11y

It should be valid as I know this syntax actually from Marcus :grin: But sure, I don't want to break any consistency...